Corona-Warn-App/src/main/res/layout/fragment_scan_check_in_qr_code.xml

```diff
@@ -28,62 +28,45 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
+        app:zxing_viewfinder_mask="@color/colorQrCodeScanMask"
         app:zxing_viewfinder_laser_visibility="false" />
 
     <TextView
         android:id="@+id/check_in_qr_code_scan_body"
-        style="@style/registrationQRCodeScanBody"
-        android:layout_width="@dimen/submission_scan_qr_code_viewfinder_size"
+        style="@style/qrCodeScanBody"
+        android:layout_width="@dimen/scan_qr_code_viewfinder_size"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/submission_scan_qr_code_viewfinder_center_offset"
-        android:text="@string/submission_qr_code_scan_body"
+        android:layout_marginTop="@dimen/scan_qr_code_viewfinder_center_offset"
+        android:text="@string/qr_code_scan_body"
         app:layout_constraintEnd_toEndOf="@id/check_in_qr_code_scan_preview"
         app:layout_constraintStart_toStartOf="@id/check_in_qr_code_scan_preview"
         app:layout_constraintTop_toBottomOf="@id/check_in_qr_code_scan_guideline_center" />
 
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/check_in_qr_code_scan_close"
-        style="@style/buttonIcon"
-        android:layout_width="@dimen/icon_size_button"
-        android:layout_height="@dimen/icon_size_button"
-        app:layout_constraintBottom_toTopOf="@id/check_in_qr_code_scan_guideline_top"
-        app:layout_constraintEnd_toStartOf="@id/guideline_start"
-        app:layout_constraintStart_toStartOf="@id/guideline_start"
-        app:layout_constraintTop_toTopOf="@id/check_in_qr_code_scan_guideline_top">
-
-        <androidx.appcompat.widget.AppCompatImageView
-            style="@style/iconStable"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:contentDescription="@string/accessibility_close"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:srcCompat="@drawable/ic_close" />
+    <com.google.android.material.appbar.MaterialToolbar
+        android:id="@+id/check_in_qr_code_scan_toolbar"
+        style="@style/CWAToolbar.BackArrow.Transparent"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:navigationIconTint="@color/colorQrCodeScanToolbar"
+        app:title="@string/qr_code_scan_title"
```

Comment:
btw the title is ellipsized not sure if it is really helpful to the user. 
<img width="373" alt="Screenshot 2021-03-29 at 06 25 35" src="https://user-images.githubusercontent.com/25054729/112786917-9c135800-9057-11eb-8413-69e0c8c2522e.png">

Reply:
qr_code_scan_title is only QR-Code Scan though right?

Reply:
in EN I it is `<string name="qr_code_scan_title">"Position the QR code in the frame."</string>`

Reply:
ah right, these need to be retranslated i think, not sure how to handle that to be honest

Reply:
@janetback Do you have any suggestions for that case?

Reply:
As the source text (german) was changed the new translation delivery (after this PR gets merged) will contain the new title in all languages.

Reply:
Hi @mtwalli - since the German has been shortened, this should automatically have an effect on all follow-on languages. The next language delivery via TEW should include the shortened texts - any manual changes to language files will be overwritten.